### ISSUE TITLE
Add support for WebTransport

### ISSUE THREAD

**Issue by DiegoRBaquero** [webtorrent/bittorrent-tracker#360]
From https://github.com/webtorrent/webtorrent/issues/2047

For reference: https://w3c.github.io/webtransport/

Better than websockets

**Comment by ThaUnknown** [comment]
is this.... really still needed? it's been 3 years and chromium is still the only place where this is supported, creating new libraries and wrappers just for chromium for this, with little benefits seems unnecessary

**Comment by DiegoRBaquero** [comment]
Needed? No, it's just a possible enhancement

**Comment by ThaUnknown** [comment]
> Needed? No, it's just a possible enhancement

*wanted

**Comment by DiegoRBaquero** [comment]
Yes, doesn't mean someone has to do it now or anything. It's just a GH issue :)

**Comment by kovalensky** [comment]
> is this.... really still needed? it's been 3 years and chromium is still the only place where this is supported, creating new libraries and wrappers just for chromium for this, with little benefits seems unnecessary

Its support [widened](https://developer.mozilla.org/docs/Web/API/WebTransport#browser_compatibility) .
